NoSQL_Schema

```diff
@@ -0,0 +1,54 @@
+
+db.addUser({
+	user: "super_admin",
+	password: password,
+	roles: [{"read","readWrite","userAdmin"}]
+})
+
+db.addUser({
+	user: "admin",
+	password: password,
+	roles: [{"read","readWrite"}]
+})
+
+db.addUser({
+	user: "super_admin",
+	password: password,
+	roles: ["read"]
+})
+
+
+user
+{
+	user_name: string,
+	password: password
```

Comment:
- Where is the role information in the schema?
- Where is the access information in the schema?